src/metaschema/oscal_control-common_metaschema.xml

```diff
@@ -143,16 +141,14 @@
                         </assembly>
                         <assembly ref="parameter-guideline" max-occurs="unbounded">
                                 <use-name>guideline</use-name>
-                                <!-- CHANGED from "guidance" to "guidelines" -->
                                 <group-as name="guidelines" in-json="ARRAY"/>
                         </assembly>
                         <choice>
                                 <field ref="parameter-value" max-occurs="unbounded">
-                                        <!-- CHANGED cardinality to allow for multiple values -->
                                         <use-name>value</use-name>
                                         <group-as name="values" in-json="ARRAY"/>
                                         <remarks>
-                                                <p>A set of values provided in a catalog can be redefined at any higher layer of OSCAL (e.g., Profile).</p>
+                                                <p>A set of values provided in a catalog can be redefined at any higher layer model of OSCAL (e.g., Profile).</p>
```

Comment:
Not a fan of the spatial "higher". Maybe just be explicit about the models where changes can occur.

Reply:
Changed it with my recommendation below per feedback in the meeting.